apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -1322,6 +1322,11 @@ expression: "&schema"
           "type": "boolean",
           "nullable": true
         },
+        "fail_open": {
```

Comment:
`required_to_start` ?

Reply:
++ I like this since `fail_open` could also apply to post-connection settings (e.g. not cause errors when inserting on entity cache updates or APQs or the like).

Reply:
fixed in  8648a87e1